.travis/linux..install.sh

```diff
@@ -16,10 +16,9 @@ fi
 
 sudo apt-get install -y $PACKAGES
 
-# Carla depends on kxstudio which creates some package conflicts (wine, etc)
-# If run too early in the dependency process.  Once provided by PPA, "carla-git"
-# can simply be added to $PACKAGES
-sudo apt-get install -y apt-transport-https software-properties-common wget
-wget https://launchpad.net/~kxstudio-debian/+archive/kxstudio/+files/kxstudio-repos_9.4.6~kxstudio1_all.deb
-sudo dpkg -i kxstudio-repos_9.4.6~kxstudio1_all.deb
+# Carla depends on kxstudio which creates some package conflicts (wine, etc) if
```

Comment:
Carla does not depend on KXStudio at all... you're just using the kxstudio repos as a shortcut.
Please change this text...